decentraland/unity-explorer
# feat: feature flags

## What does this PR change?

Adds the implementation for fetching the feature flags configuration from: https://features.decentraland.systems/.
It supports:
- Send the current user address
- Set the app name (explorer)
- Set the hostname (decentraland.org)
- Variants: get the payload as json, csv or text
- Configure the fetch url via `--feature-flags-url` program argument. Example: `./Decentraland.app --feature-flags-url https://feature-flags.decentraland.zone`

## How to test the changes?

1. Launch the explorer
2. Check that the startup of the app works as expected

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
